lib/working_hours/computation.rb

```diff
@@ -85,6 +85,32 @@ def advance_to_working_time time, config: nil
       end
     end
 
+    def next_working_time time, config: nil
+      config ||= wh_config
+      time = in_config_zone(time, config: config).round
+      loop do
+        # skip holidays and weekends
+        while not working_day?(time, config: config)
+          time = (time + 1.day).beginning_of_day
+        end
+        # find next working range after time
+        time_in_day = time.seconds_since_midnight
+        time = time.beginning_of_day
```

Comment:
I thought I got that covered in a test below (line 176), or am I missing something? If you can give me a pointer I'll get it fixed.